doc/source/user_guide/migration.rst

```diff
@@ -0,0 +1,215 @@
+-------------------------------
+
+You can customize the backend of the Ansys Tools Visualization Interface plotter to enable or turn off certain functionalities. For example,
+if you want to enable picking, you can do it as follows:
+
+.. code-block:: python
+
+    from ansys.tools.visualization_interface import Plotter
+    from ansys.tools.visualization_interface.backends import PyVistaBackend
+
+    backend = PyVistaBackend(allow_picking=True)
+
+    # Create a plotter
+    pl = Plotter(backend=backend)
+
+    # Add the MeshObjectPlot instance to the plotter
+    pl.plot(mesh_object_plot)
+
+    # Show the plotter
+    pl.show()
+
+If you want to go further and customize the backend even more, you can create your own backend by inheriting from the ``PyVistaBackendInterface`` class
+and implementing the required methods. You can find more information about this in the backend documentation:
+
+.. code-block:: python
+
+    @abstractmethod
+    def plot_iter(self, plottable_object: Any, name_filter: str = None, **plotting_options):
+        """Plot one or more compatible objects to the plotter.
+
+        Parameters
+        ----------
+        plottable_object : Any
+            One or more objects to add.
+        name_filter : str, default: None.
+            Regular expression with the desired name or names  to include in the plotter.
+        **plotting_options : dict, default: None
+            Keyword arguments. For allowable keyword arguments, see the
+            :meth:`Plotter.add_mesh <pyvista.Plotter.add_mesh>` method.
+
+        """
+        pass
+
+
+    @abstractmethod
+    def plot(self, plottable_object: Any, name_filter: str = None, **plotting_options):
+        """Plot a single object to the plotter.
+
+        Parameters
+        ----------
+        plottable_object : Any
+            Object to add.
+        name_filter : str
+            Regular expression with the desired name or names to include in the plotter.
+        **plotting_options : dict, default: None
+            Keyword arguments. For allowable keyword arguments, see the
+            :meth:`Plotter.add_mesh <pyvista.Plotter.add_mesh>` method.
+
+        """
+        pass
+
+
+The rest of the methods are implemented for you. This ensures that while you can customize what you need for plotting, the rest of the functionalities still work as expected.
+If you need to even go further, you can also create your own plotter by inheriting from the ``BaseBackend`` class and implementing the required methods,
+although this may break existing features. You can find more information about this in the plotter documentation.
+
+Customize the picker or hover behavior
+--------------------------------------
+You can customize the picker of the Ansys Tools Visualization Interface plotter to decide what happens when an object is picked or hovered.
+For example, if you want to print the name of the picked object, you can do it as described in the custom picker example.
```

Comment:
Again, it would be nice to have this sentence be something like this:
"... you can do it as described in the X example..." where you link to the correct example.